docs/submarine-sdk/README.md

```diff
@@ -1,50 +0,0 @@
-<!---
-  Licensed under the Apache License, Version 2.0 (the "License");
-  you may not use this file except in compliance with the License.
-  You may obtain a copy of the License at
-
-   http://www.apache.org/licenses/LICENSE-2.0
-
-  Unless required by applicable law or agreed to in writing, software
-  distributed under the License is distributed on an "AS IS" BASIS,
-  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  See the License for the specific language governing permissions and
-  limitations under the License. See accompanying LICENSE file.
--->
-
-# Submarine-SDK
```

Comment:
I think we still need `docs/submarine-sdk/README.md`, ‘
You can change `README.md` context to jump index.
Just like: https://github.com/apache/hadoop-submarine/blob/master/docs/README.md

Reply:
Thanks @liuxunorg for the review and comment, i'm going to update it

Reply:
I add a README for submarine-sdk temporarily, but we need to add more description after other features complete.